docs/sources/flow/reference/components/loki.process.md

```diff
@@ -308,25 +312,15 @@ Name          | Type      | Description
 `source`      | `string`  | Name from extracted data to parse. If empty, uses the log message.  | `""`    | no
 
 
-The `expression` field needs to be a Go RE2 regex string. Every capture group (re) is set into the extracted map, so it must be named like: `(?P<name>re)`. The name of the capture group is then used as the key in the extracted map.
+The `expression` field needs to be a RE2 regex string. Every capture group (re)
+is set into the extracted map, so it must be named like: `(?P<name>re)`. The
+name of the capture group is then used as the key in the extracted map.
 
-<!--
-We don't care about YAML, what does River do instead???
+Because of how River treats backslashes in double-quoted strings, all
+backslashes in an `expression` must be escaped like `"\\S+"`.
```

Comment:
I've removed the reference to 'double-quoted' strings, as River only has that kind. How does that read now?

```
Because of how River strings work, any backslashes in `expression` must be
formatted as a double backslash; for example `"\\w"` or `"\\S+"`.
```

Reply:
Looks good